core/external/L.Geodesic.js

```diff
@@ -8,42 +8,40 @@
   // as north/south lines have very little curvature in the projection, we can use longitude (east/west) seperation
   // to calculate intermediate points. hopefully this will avoid the rounding issues seen in the full intermediate
   // points code that have been seen
-  function geodesicConvertLine (start, end, convertedPoints) {
+  function geodesicConvertLine (start, end, convertedPoints) { // push intermediate points into convertedPoints
 
     var lng1 = start.lng * d2r;
     var lng2 = end.lng * d2r;
     var dLng = lng1-lng2;
 
     var segments = Math.floor(Math.abs(dLng * earthR / this.options.segmentsCoeff));
```

Comment:
Please change this to Math.ceil( ). see "geodesic" discussion

Reply:
That is not subject of this PR.